### ISSUE TITLE
Add `langChanged$` observable

### ISSUE THREAD

**Issue by MickL** [ngneat/transloco#461]
### Is your feature request related to a problem? Please describe.
TranslocoService provides an observable that emits when the language is about to change. But there is no observable that emits when the language has changed.

### Current behavior
When using `langChanges$` to trigger rerendering a pipe the pipe rerenders but the language has not been loaded yet and therefor the pipe rerenders in the same language as before. This would be useful for https://github.com/ngneat/transloco/issues/443

**Comment by shaharkazaz** [comment]
You can use events: https://ngneat.github.io/transloco/docs/translation-api/#events

**Comment by MickL** [comment]
This will not work, please reopen. From the docs:

> Note that these events will only fire when the translation is loaded from the server, meaning that when switching from a given lang and back won't trigger the events again since the language is loaded from the cache.

**Comment by mfodor** [comment]
I face with the very same problem in our project as well. I've already tried with `events$` which can help when the first lang change happens, but it is not fired after that.

I need to translate computed keys programmatically in the controller. So this is working fine now:

```typescript
    this.transloco.langChanges$.subscribe(() => {
      this.buildHRParams();

      // this is needed to fix translation when the other language is loaded for the first time and so is not available
      // but is not enough since this is never called again later
      this.transloco.events$.pipe(
        filter(e => e.type === 'translationLoadSuccess'),
        pluck('payload'),
        first()
      ).subscribe(() => {
        this.buildHRParams();
      });
    });
```

But is this really a good API to use? Why should I call `buildHRParams()` twice? Why should I subscribe in a subscription for the (kind of the) *same* event?

`langChanges$` is fired before loading the other language file  
`events$` fired only when real loading happened and not the cache has been used. We could say that it's "bug" should be fixed and fire a `translationLoadSuccess` event even when loaded from the cache but I'd still say that why should I do 2 subscription for the 1 event I am interested of.

It's easy to see that there is a valid need to have a `langChanged$` event which is fired always after having the new translations available.

**Comment by MickL** [comment]
I dont see a use case for having an event that emits BEFORE the language has changed as well but an urgent need for an event AFTER the language has changed.

**Comment by gabriel-gn** [comment]
Is it possible to make the "events$" observable to emit a new event like 'langChanged' when it is triggered?

**Comment by gabriel-gn** [comment]
@shaharkazaz is it possible to open this issue again? It's a nice addition to the library

**Comment by MickL** [comment]
I dont know why this has been closed. It is an essential feature to have.

**Comment by shaharkazaz** [comment]
@MickL @gabriel-gn You are welcome to open a PR

**Comment by gabriel-gn** [comment]
> @MickL @gabriel-gn You are welcome to open a PR

will try asap

**Comment by gabriel-gn** [comment]
Added PR https://github.com/ngneat/transloco/pull/534

**Comment by mfodor** [comment]
@shaharkazaz 
Are there any plans or roadmap when will this be merged?

**Comment by costalvaro** [comment]
Upvoting this issue, a `langChanged$` observable would definitely help properly handling language switching.

**Comment by parker-mike** [comment]
@gabriel-gn, In order for everyone to benefit from your PR, please, correct the commit messages so they follow the [guidelines](https://github.com/ngneat/transloco/blob/master/CONTRIBUTING.md#commit).
feat: Add langChanged$ observable

fixes [#461](https://github.com/ngneat/transloco/issues/461)

PR Close [#534](https://github.com/ngneat/transloco/pull/534)

docs: Add langChanged$ observable
test: Add langChanged$ observable

**Comment by shaharkazaz** [comment]
Available in v3.2.0.
